### PR TITLE
Crs 109 sentence calculation object

### PR DESCRIPTION
**Pull request by joelstobart-moj**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/OffenderSentenceProfileCalculation.kt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model
+
+import java.time.LocalDate
+
+data class OffenderSentenceProfileCalculation(
+  var licenceExpiryDate: LocalDate?,
+  var sentenceExpiryDate: LocalDate?,
+  var releaseDate: LocalDate?,
+  var topUpSupervisionDate: LocalDate?,
```
Comment (yasinmustafa) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/OffenderSentenceProfileCalculation.kt:9]: is this the same thing as the _post sentence supervision expiry date_?

Comment (joelstobart-moj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/OffenderSentenceProfileCalculation.kt:9]: It is :)

Comment (joelstobart-moj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/OffenderSentenceProfileCalculation.kt:9]: We should ask louise her preference

**Review by yasinmustafa — APPROVED** [review]
LGTM - just one Q

**Pull request merged**